polars/io.ts

```diff
@@ -476,33 +476,56 @@ interface RowCount {
 }
 
 interface ScanParquetOptions {
-  columns?: string[] | number[];
-  numRows?: number;
+  nRows?: number;
```

Comment:
i prefer `numRows` here.

Reply:
I changed it to make it consistent with the Rust and Python versions which have `n_rows`